src/tools/genpolicy/update_policy_samples.py

```diff
@@ -58,6 +58,16 @@ def timeRunCmd(arg):
 print("========== COMMAND: " + next_command)
 runCmd(next_command)
 
+print("Modifying settings for testing")
+runCmd("cp genpolicy-settings.json default-genpolicy-settings.json")
+
+# todo: extract jq logic below to adapt_settings.sh and use that as a way to keep changes in sync with MAP
```

Comment:
I opt for addressing this todo as part of this PR

Reply:
addressed